faiss/impl/ProductQuantizer.cpp

```diff
@@ -60,7 +60,11 @@ void ProductQuantizer::set_derived_values() {
             d % M == 0,
             "The dimension of the vector (d) should be a multiple of the number of subquantizers (M)");
     dsub = d / M;
-    code_size = (nbits * M + 7) / 8;
+    if (nbits > 0)
```

Comment:
please add FAISS_THROW_IF_NOT(nbits > 0)

Reply:
ok  I update it

Reply:
I add  FAISS_THROW_IF_NOT(nbits > 0)

some checks error.

such as :

RuntimeError: Error in void faiss::ProductQuantizer::set_derived_values() at /home/circleci/miniconda/conda-bld/faiss-pkg_1698213843309/work/faiss/impl/ProductQuantizer.cpp:64: Error: 'nbits > 0' failed